app/controllers/admin/api/services_controller.rb

```diff
@@ -12,7 +12,7 @@ class Admin::Api::ServicesController < Admin::Api::ServiceBaseController
   # Service List
   # GET /admin/api/services.xml
   def index
-    services = accessible_services.includes(:proxy, :account, :annotations).order(:id).paginate(pagination_params)
+    services = accessible_services.includes(:proxy, :annotations).order(:id).paginate(pagination_params)
```

Comment:
I wonder how it worked before :thinking: but so be it.

Reply:
Yeah, no idea why it broke with the upgrade :shrug: 
Could be due to changes either in rails or in bullet that also had to be upgraded. Maybe it was not identifying such issues before.

Reply:
Or a third option - that account not being preloaded was detected before but not anymore :)

Lets hope for the best.